components/widget/widget_table.lua

```diff
@@ -0,0 +1,61 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Widget/Table
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Widget = require('Module:Infobox/Widget')
+local WidgetFactory = require('Module:Infobox/Widget/Factory')
+
+local Table = Class.new(
+	Widget,
+	function(self, input)
+		self.rows = input.rows or {}
+		self.classes = input.classes or {}
+		self.css = input.css or {}
+	end
+)
+
+function Table:addRow(row)
+	table.insert(self.rows, row)
+	return self
+end
+
+function Table:addClass(class)
+	table.insert(self.classes, class)
+	return self
+end
+
+function Table:make()
+	local table = mw.html.create('div')
```

Comment:
i suggest to not name it "table" as it would overwrite the normal table variable (table.insert etc. wouldn't work anymore

Reply:
Renamed